.github/workflows/analyze-test.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Setup flutter
         uses: subosito/flutter-action@v2
         with:
-          flutter-version: "3.16.0"
+          flutter-version: "3.22.2"
```

Comment:
Also upgrade flutter-version in release mode on CI

Reply:
Done